validator/endpoints/tasks.py

```diff
@@ -203,18 +204,52 @@ async def get_leaderboard(
 
 async def get_network_status(
     config: Config = Depends(get_config),
-) -> TrainingTaskStatus:
+) -> NetworkStats:
     try:
         logger.debug("IN get network status")
-        training_stats = await task_sql.get_training_tasks_stats(config.psql_db)
-        if training_stats.number_of_jobs_training >= MAX_CONCURRENT_JOBS:
-            training_stats.job_can_be_made = False
-        return training_stats
+        stats = await task_sql.get_current_task_stats(config.psql_db)
+        if stats.number_of_jobs_training >= MAX_CONCURRENT_JOBS:
+            stats.job_can_be_made = False
+        return stats
     except Exception as e:
         logger.info(f"There was an issue with getting training status {e}")
         raise HTTPException(status_code=500, detail=str(e))
 
 
+async def get_completed_organic_tasks(
+    hours: int | None = None,
+    config: Config = Depends(get_config),
+) -> List[TaskDetails]:
+    """Get all completed organic tasks from the last N hours"""
+    lookup_hours = hours if hours is not None else config.default_organic_task_history_hours
```

Comment:
should be a default directly in the args, with pydantic field or query so you can explain what the param is directly as well